OPT_mission.Tanoa/sectorcontrol/functions/fnc_setupflag.sqf

```diff
@@ -22,15 +22,15 @@
 private _maxTries = 100;
 while {(count GVARMAIN(nato_flags) < round OPT_sectorcontrol_flagCountNATO) && _maxTries > 0} do
 {
-    GVARMAIN(nato_flags) pushBackUnique [selectRandom (allMissionObjects "FlagPole_F" select {(_x getVariable "owner" == west) || (_x getVariable "owner" == sideUnknown)})];
+    GVARMAIN(nato_flags) pushBackUnique selectRandom (allMissionObjects "FlagPole_F" select {(_x getVariable "owner" == west) || (_x getVariable "owner" == sideUnknown)});
```

Comment:
Hier würde ich `allMissionObjects "FlagPole_F"` vor der Schleife in eine Variable speichern, damit das nicht bei jeder Iteration wieder neu bestimmt werden muss. Das ändert sich ja nicht.
Wenn sich die `owner`-Variable auch nicht ändert, kann man sogar das gesamte Statement mit samt dem `select` vor die Schleife ziehen und zwischenspeichern.